src/Core/Application/Catalog/Products/ProductExportDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FSH.WebApi.Application.Catalog.Products;
+
+public class ProductExportDto : IDto
+{
+    public string Name { get; set; } = default!;
+    public string Description { get; set; } = default!;
```

Comment:
Description should probably be nullable?

Reply:
It should be empty string if null.

Reply:
All the dto's until now are consistent wrt their nullability with the business objects they represent... and Description is nullable on Product...

Reply:
The Excel exporter cannot take null values. So If null they must be empty string.

Reply:
Then you should actually assign it by default with string.Empty instead of "default"... because now if the description is null in the database it will be null here!